ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -54,9 +55,12 @@ export const DEFAULT_LINE_WIDTH = 1.5;
 
 export const DEFAULT_POINT_RADIUS = 4;
 
+export const DEFAULT_AREA_OPACITY = 0;
+
 export const DEFAULT_VISUAL: VisualOptions = {
   line_width: DEFAULT_LINE_WIDTH,
   point_radius: DEFAULT_POINT_RADIUS,
+  area_opacity: DEFAULT_AREA_OPACITY,
```

Comment:
I started adding a new visual regression test for this new option and other visual / y_axis options, but I need to follow up with @juliepagano about a question with mocking the range query response, we may need to adjust some the types:

![image](https://user-images.githubusercontent.com/9369625/219688311-823e2c48-1cd7-4940-99db-3220f89c1f76.png)

Reply:
FYI for others that we chatted about this offline. Lemme know if you need any extra help on this when you do it in a follow-up.